### ISSUE TITLE
Campfire should use unaltered item model for non-block items

### ISSUE THREAD

**Issue by Exsolutus** [codetaylor/pyrotech#122]
The way the campfire scales the models/textures for fuels works well for blocks as fuel. However, non-block item models can look strange because of this. Example attached.

![2019-07-12_12 58 42](https://user-images.githubusercontent.com/23705866/61145206-d9f7d380-a4a4-11e9-956f-0eb9740dcc31.png)

Non-block items should not receive the model scaling when used in the campfire, and instead use the existing item model.

**Comment by codetaylor** [comment]
Yeah, this is currently a limitation of the interaction framework that was developed for the mod.
